src/app/beer_garden/events/__init__.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+import logging
+from datetime import datetime, timezone
+
+import wrapt
+from brewtils.models import Event, Events
+
+from beer_garden import config as config
+
+# In this master process this should be an instance of EventManager, and in entry points
+# it should be an instance of EntryPointManager
+manager = None
+
+logger = logging.getLogger(__name__)
+
+
+def publish(event: Event) -> None:
+    """Convenience method for publishing events
+
+    All this does is place the event on the queue for the process-wide manager to pick
+    up and process.
+
+    Args:
+        event: The event to publish
+
+    Returns:
+        None
+    """
+    # Do some formatting / tweaking
+    if not event.garden:
+        event.garden = config.get("garden.name")
```

Comment:
Chat with me about how the ```event.garden``` field would get populated outside of this. If this is only being used to send events from child to parent, then I don't see a problem with setting it, but we need to make sure we don't have any logic later in the process that looks up that garden_name. Anything past child relationships is obfuscated to the parent because they shouldn't care about grandchildren.